code/game/objects/structures/rlattice.dm

```diff
@@ -0,0 +1,7 @@
+/obj/structure/lattice/rlattice
+	name = "reinforced lattice"
+	desc = "A support lattice. theese ones are reinforced to widstand extreeme enviroments." //cosmetic latice for all of your industrial themes in lava or what not
```

Comment:
>  theese ones are reinforced to widstand extreeme enviroments

these* / withstand* / extreme*

Reply:
i'll... fix em alongside the piping&wire issues you described